src/hpr_config_worker.erl

```diff
@@ -114,11 +131,15 @@ handle_continue(
             ok = process_route_stream_res(hpr_route_stream_res:from_map(RouteStreamRes), Path),
             {noreply, State, {continue, ?RCV_CFG_UPDATE}};
         eof ->
-            lager:warning("got eof"),
-            _ = grpc_client:stop_connection(Connection),
-            {noreply, State, {continue, ?CONNECT}};
+            {Delay, Backoff1} = backoff:fail(Backoff0),
```

Comment:
After this `backoff:fail/1`, we go back to `handle_continue(?CONNECT, ...)`, which means the connect is now starting with a longer backoff. I'm wondering if the calls to `backoff` are needed here. Since we don't stay in `RCV_CFG_UPDATE` after the failure, do we need to call `backoff:succeed/1`?